src/components/footer/Pager.ts

```diff
@@ -112,7 +112,7 @@ export class DataTablePager {
 
       this.onPaged.emit({
         type: 'pager-event',
-        value: page
+        page: page
```

Comment:
Need to change this then
` onPageChanged(action) {
    this.state.setPage(this.key, action);
    this.onPageChange.emit(action.page);
  }
`
since it is receiving the event and has no page property.